lib/mongo/srv/records.rb

```diff
@@ -0,0 +1,72 @@
+# Copyright (C) 2014-2019 MongoDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'resolv'
+
+module Mongo
+
+  module SRV
```

Comment:
Since `Mongo::SRV` is not an existing namespace, the new namespace should be called `Mongo::Srv`.

Reply:
I think it's confusing to have "SRV" in different case in two different namespaces in the driver, but ok